### PR TITLE
fix: Edit runbook agent

### PR DESCRIPTION
**Pull request by adityachoudhari26**

### CODE REVIEW

File: apps/webservice/src/app/[workspaceSlug]/systems/[systemSlug]/deployments/[deploymentSlug]/page.tsx [hunk 1]
```diff
@@ -130,7 +130,16 @@ export default async function DeploymentPage({
                       value={deployment.jobAgentConfig}
                       jobAgents={jobAgents}
                       workspace={workspace}
-                      deploymentId={deployment.id}
+                      onSubmit={async (data) => {
```
Comment (adityachoudhari26) [inline apps/webservice/src/app/[workspaceSlug]/systems/[systemSlug]/deployments/[deploymentSlug]/page.tsx:133]: I tried doing without the brackets but it became less redable so just kept it with the async await

File: packages/api/src/router/deployment.ts [hunk 2]
```diff
@@ -119,38 +114,7 @@ export const deploymentRouter = createTRPCRouter({
...
-                      releaseJobTriggers.map((jc) => jc.release_job_trigger),
-                    )
-                    .filter(isPassingAllPolicies)
-                    .then(cancelOldReleaseJobTriggersOnJobDispatch)
-                    .dispatch()
-                    .then(() => d),
-                )
-            : d,
-        ),
+        .where(eq(deployment.id, input.id)),
```
Comment (adityachoudhari26) [inline packages/api/src/router/deployment.ts:117]: removed per discussion

File: apps/webservice/src/app/[workspaceSlug]/systems/[systemSlug]/runbooks/RunbookDropdownMenuItem.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+import { DropdownMenuItem } from "@ctrlplane/ui/dropdown-menu";
+
+export const RunbookDropdownMenuItem = forwardRef<
+  HTMLDivElement,
+  React.ComponentPropsWithoutRef<typeof DropdownMenuItem>
+>((props, ref) => (
+  <DropdownMenuItem {...props} ref={ref} onSelect={(e) => e.preventDefault()}>
+    {props.children}
+  </DropdownMenuItem>
+));
```
Comment (jsbroks) [inline apps/webservice/src/app/[workspaceSlug]/systems/[systemSlug]/runbooks/RunbookDropdownMenuItem.tsx:14]: I won't create a whole new component just for this

File: apps/webservice/src/app/[workspaceSlug]/systems/[systemSlug]/runbooks/page.tsx [hunk 4]
```diff
@@ -9,9 +9,16 @@ import { RunbookRow } from "./RunbookRow";
...
+export default async function Runbooks({
+  params,
+}: {
+  params: { workspaceSlug: string; systemSlug: string };
+}) {
+  const workspace = await api.workspace.bySlug(params.workspaceSlug);
+  if (workspace == null) return notFound();
   const system = await api.system.bySlug(params).catch(notFound);
   const runbooks = await api.runbook.bySystemId(system.id);
+  const jobAgents = await api.job.agent.byWorkspaceId(workspace.id);
```
Comment (jsbroks) [inline apps/webservice/src/app/[workspaceSlug]/systems/[systemSlug]/runbooks/page.tsx:21]: include the job agent as a innerjoin on the runbook query?

File: packages/auth/src/utils/rbac.ts [hunk 5]
```diff
@@ -215,6 +199,22 @@ const getDeploymentScopes = async (id: string) => {
   ];
 };
 
+const getRunbookScopes = async (id: string) => {
```
Comment (jsbroks) [inline packages/auth/src/utils/rbac.ts:202]: we already have this logic?

https://github.com/sizzldev/ctrlplane/blob/ba12569a384c755edbe8de69e47cadcb9e093292/packages/auth/src/utils/rbac.ts#L96

**Pull request merged**